packages/server/graphql/getRateLimiter.ts

```diff
@@ -1,11 +1,41 @@
-import RateLimiter from './RateLimiter'
+import {InMemoryRateLimiter, StubRateLimiter, RateLimiter} from '../utils/rateLimiters'
+
+const RATE_LIMITERS = Object.freeze({
+  'in-memory': InMemoryRateLimiter,
+  stub: StubRateLimiter
+})
 
 let rateLimiter: RateLimiter
-const getRateLimiter = () => {
+export default function getRateLimiter() {
   if (!rateLimiter) {
-    rateLimiter = new RateLimiter()
+    rateLimiter = _getRateLimiter()
   }
   return rateLimiter
 }
 
-export default getRateLimiter
+/**
+ * This function returns the appropriate rate limited based on the RATE_LIMITER
+ * environment variable, falling back to the InMemoryRateLimiter (the previous,
+ * uncustomizable default).
+ *
+ * Exposed for testing. Use the default export outside of test.
+ *
+ * @private
+ */
+export function _getRateLimiter(): RateLimiter {
+  const rateLimiterEnvVar = process.env.RATE_LIMITER
```

Comment:
Plug for my friend Stephen's `ts-app-env` project: https://github.com/stephenh/ts-app-env/. This makes dealing with environment variables really easy!